README.rst

```diff
@@ -182,3 +182,13 @@ Notes
 
 * In order for the algorithm to work the mesh must not be numbered incorrectly
   or disconnected or of somehow degenerate.
+
+* The c++ compiler must have OpenMP installed. This is generally not an issue
+  on g++ or Microsoft Visual Studio. However, in macOS one may need to install
+  llvm from brew in order to use OpenMP: ``brew install llvm``. Then, change the
+  ``CC`` and ``CXX`` environment variables to the following:
```

Comment:
What is happening when openMP is missing? Can we still use tvb-gdist?
Aren't the openMP pragmas simply ignored?

Reply:
They are ignored in g++ and MSVC but not in clang++.